migrations/versions/0226_new_letter_rates.py

```diff
@@ -0,0 +1,54 @@
+"""empty message
+
+Revision ID: 0226_new_letter_rates
+Revises: 0225_another_letter_org
+
+"""
+
+revision = '0226_new_letter_rates'
+down_revision = '0225_another_letter_org'
+
+import uuid
+import pytz
+from datetime import datetime
+from alembic import op
+
+
+start = datetime(2018, 10, 1, 0, 0, tzinfo=pytz.utc)
```

Comment:
I think we can get away without adding in timezone info - we don't do this elsewhere, we just keep the database in UTC.

Reply:
That's right, the database is timezone agnostic. It doesn't persist which timezone you use. This means we need persist the notification as UTC. October first is still in BST, start will be persisted as 2018-10-01 00:00:00, however we actually need it to start 2018-09-30 23:00:00. In short if you create `start = datetime(2018, 09, 30, 23, 0)` that should be fine.

Reply:
ok, cheers, will do :)